platforms/shared/configuration/site.yaml

```diff
@@ -32,6 +32,8 @@
 # Reseting the network
   - import_playbook: "{{ playbook_dir }}/../../r3-corda/configuration/reset-network.yaml"
     when: network.type == 'corda' and reset is defined and reset == 'true'
+  - import_playbook: "{{ playbook_dir }}/../../corda-ent/configuration/reset-network.yaml"
```

Comment:
Already added.

Reply:
Fixed in [ae9a768](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/992/commits/ae9a76899ddcce9ea9e2067c3ba79ebc784c08c5); duplicated on merge from `upstream/develop`